### PR TITLE
chore(ci) remove travis-ci

### PR DESCRIPTION
**Pull request by hutchic**
we've ran travis and github actions in parallel for a sufficiently long enough time to be confident in GH Actions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +13,7 @@ from startups to Global 5000 as well as government organizations.
 
 [Installation](https://konghq.com/install) |
 [Documentation](https://docs.konghq.com) |
-[Forum](https://discuss.konghq.com) |
+[Forum](https://discuss.konghq.com) |mmi
```
Comment (eskerda) [inline README.md:16]: 😕

**Comment by kidd** [comment]
Are these gh actions that currently run kong tests able to run the enterprise edition test suite? EE should have a way to run tests in place when this merge cascades to EE.

**Comment by hutchic** [comment]
@kidd at this time Kong EE hasn't been migrated to KBT which GHA is using as the piping to run the tests. There's been some preliminary work done in that vein but never a concerted effort to wrap up the remaining work.

Options:
- make KBT kong-ee compatible and migrate both Kong / Kong Enterprise fully to GHA (preferable)
- make KBT kong-ee compatible and migrate both Kong (back to) travis-ci
- turn off GHA for Kong Enterprise and continue to run tests via travis-ci (quickest)

**Comment by gszr** [comment]
Travis CI was disabled in the UI instead. Closing this for now.

**Pull request closed without merging**